packages/worksheet/src/components/WorksheetBox/index.js

```diff
@@ -0,0 +1,222 @@
+import { IconByName } from "@shiksha/common-lib";
+import {
+  Avatar,
+  Box,
+  HStack,
+  Pressable,
+  Stack,
+  Text,
+  VStack,
+} from "native-base";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { useNavigate } from "react-router-dom";
+// import {
+//   LinkedinShareButton,
+//   LinkedinIcon,
+//   WhatsappIcon,
+//   WhatsappShareButton,
+// } from "react-share";
+
+export default function WorksheetBox({ item, url, canShare, _addIconButton }) {
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+  const colors = ["lightBlue.800", "indigo.900", "fuchsia.700", "rose.600"];
+  const random = Math.floor(Math.random() * (4 - 1) + 1) - 1;
+
+  return (
+    <Box p="5" borderWidth="1" borderColor="gray.300" rounded="lg">
+      <VStack space={4}>
+        <HStack justifyContent="space-between" alignItems="flex-start">
+          <Pressable onPress={() => (url ? navigate(url) : "")}>
+            <HStack space={2} alignItems="center">
+              <Avatar bg={colors[random]} size="57" rounded="md">
+                <Text fontWeight="600" fontSize="16px" color="white">
+                  {item.heading?.toUpperCase().substr(0, 1)}
+                </Text>
+              </Avatar>
+              <Stack space="1">
+                <VStack space="1px">
+                  <Text fontWeight="600" fontSize="16px">
+                    {item.heading}
+                  </Text>
+                  {/*<HStack space={1} alignItems="center">
+                     <Text fontWeight="400" fontSize="10px">
+                      {item.subHeading} {"•"}
+                    </Text> 
+                    <Text fontWeight="400" fontSize="10px">
+                      {t("CLASS") + " " + item.class}
+                    </Text>
+                    
+                  </HStack>*/}
+                </VStack>
+
+                <HStack space={1} alignItems="center">
+                  <IconByName
+                    name="Heart3FillIcon"
+                    color="red.500"
+                    _icon={{ size: 12 }}
+                    isDisabled
+                  />
+                  <Text fontWeight="600" fontSize="10px">
+                    {item.likes + " likes"}
+                  </Text>
```

Comment:
Use text types.

Reply:
how?

Reply:
that is in another PR https://github.com/shiksha-platform/frontend-modulefederation/pull/123